projects/angular/src/forms/combobox/_combobox.clarity.scss

```diff
@@ -369,53 +191,39 @@
 
     cds-icon[shape='angle'],
     clr-icon[shape='angle'] {
-      @include mixins.css-var(
-        color,
-        clr-combobox-trigger-font-color,
-        combobox-variables.$clr-combobox-pill-font-color,
-        variables.$clr-use-custom-properties
-      );
+      color: combobox-variables.$clr-combobox-trigger-font-color;
 
-      @include mixins.min-equilateral-var(
-        clr-combobox-caret-icon-size,
-        variables.$clr_baselineRem_12px,
-        variables.$clr-use-custom-properties
-      );
+      @include mixins.min-equilateral(clr-combobox-caret-icon-size);
     }
   }
 
   .clr-combobox-options {
     @include menu-mixins.generate-dropdown-menu();
     position: static;
 
-    max-height: variables.$clr_baselineRem_10;
+    max-height: mixins.baselinePx(240);
     overflow-y: scroll;
 
     visibility: visible;
 
     .clr-combobox-options-loading {
-      padding: variables.$clr_baselineRem_0_125 variables.$clr_baselineRem_1;
+      padding: mixins.baselinePx(3) tokens.$cds-global-space-1;
```

Comment:
For refactoring ticket. Paddings should ne divisible by 2.

Reply:
Same as previous.